### PR TITLE
tests: split compose sanity tests and add more. Refs #356, #615

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: cmd/osbuild-tests/main_test.go [hunk 1]
```diff
@@ -18,24 +18,48 @@ import (
...
+
+	runComposerCLI(t, false, "compose", "logs", uuid.String())
+	_, err = os.Stat(uuid.String() + "-logs.tar")
+	require.NoError(t, err, "'%s-logs.tar' not found", uuid.String())
+	defer os.Remove(uuid.String() + "-logs.tar")
+
+	// runComposerCLI(t, false, "compose", "cancel", UUID)
+	runComposerCLI(t, false, "compose", "image", uuid.String())
+	_, err = os.Stat(uuid.String() + ".ami")
+	require.NoError(t, err, "'%s.ami' not found", uuid.String())
```
Comment (atodorov) [inline cmd/osbuild-tests/main_test.go:58]: @major I see:

```
skipping: [localhost] => (item={'name': 'osbuild-tests', 'timeout': 30}) 
```

and the above piece of code is wrong. The filename is `<uuid>-image.vhdx` now. Do you know why Schutzbot skipped this test ?

Comment (atodorov) [reply cmd/osbuild-tests/main_test.go:58]: Nevermind, I found the failure. I've been looking at the output of another job.

**Comment by atodorov** [comment]
```
# composer-cli compose logs 56621ea8-de30-41d2-bf78-32aa3cde0669
Opening log for compose 56621ea8-de30-41d2-bf78-32aa3cde0669 failed
```

and the sanity test on line 68 fails.

@ondrejbudai ^^^ is this related to #617  ?

**Comment by ondrejbudai** [comment]
Yeah, it is

**Pull request merged**